docs/tutorials/general.md

```diff
@@ -0,0 +1,350 @@
+# OASGraph General tutorial
+
+The purpose of this tutorial is to give an in-depth look into OASGraph.
```

Comment:
Let's be specific and add here that this is an in-depth look into using OASGraph together with LoopBack v4.

Reply:
Ok, I was trying to say a more in-depth look into OASGraph than the quick start guide but this is probably better.